eo-runtime/src/main/eo/org/eolang/bytes.eo

```diff
@@ -22,7 +22,7 @@
 
 +package org.eolang
 +home https://github.com/objectionary/eo
-+version 0.0.0
++version 0.1.0
```

Comment:
The policy is not to update the version manually.
It should always be `0.0.0` in master and development branches.
It's incremented automatically during deployment process.